tests/models/vilt/test_modeling_vilt.py

```diff
@@ -224,7 +226,7 @@ def _prepare_for_class(self, inputs_dict, model_class, return_labels=False):
                 inputs_dict["labels"] = torch.zeros(
                     self.model_tester.batch_size, self.model_tester.num_labels, device=torch_device
                 )
-            elif model_class.__name__ == "ViltForMaskedLM":
+            elif model_class.__name__ in ["ViltForMaskedLM", "ViltForTokenClassification"]:
```

Comment:
Can you also remove lines 221 and 222 which are commented out?

Also, can you add a `create_and_check_for_token_classification` method (similar to `create_and_check_model`), with a corresponding `test_for_token_classification` method (similar to `test_model`)?

Reply:
@NielsRogge - Hi, I removed the commented out lines and added the new test test_for_token_classification.

Reply:
Awesome!